swap_meet/vendor.py

```diff
@@ -1,2 +1,84 @@
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory == None else inventory 
+    
+    def add(self, item):
+        self.inventory.append(item)
+        
+        return item
+    
+    def remove(self, item):
+        result = False
+
+        if item in self.inventory:
+            self.inventory.remove(item)
+            result = item
+
+        return result
+    
+    def get_by_id(self, item_id):
+        results = None
+
+        for item in self.inventory:
+            if item.id == item_id:
+                results = item
+                break
+
+        return results
```

Comment:
Instead of using the `break` keyword you could just opt for the `return`, once you find the item you are looking for you could exit out of the function since there is no more logic to run.